easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-foss-2018b-Perl-5.28.0.eb

```diff
@@ -1,23 +1,27 @@
+# easybuild easyconfig
 easyblock = 'ConfigureMake'
 
 name = 'VCFtools'
 version = '0.1.16'
 versionsuffix = '-Perl-%(perlver)s'
 
 homepage = "https://vcftools.github.io"
-description = """The aim of VCFtools is to provide 
- easily accessible methods for working with complex 
+description = """The aim of VCFtools is to provide
+ easily accessible methods for working with complex
  genetic variation data in the form of VCF files."""
 
 toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': True}
```

Comment:
@fizwit `optarch` is enabled by default, so this doesn't make much sense, please remove it